grunt_tasks/fontello.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Define tasks that download and copy fontello font files.
+ */
+module.exports = function (grunt) {
+    var path = require('path');
+
+    grunt.config.merge({
+        curl: {
+            fontello: {
+                src: 'https://data.kitware.com/api/v1/file/57c5d1fc8d777f10f269dece/download',
```

Comment:
I consider this a temporary solution until the network problems are resolved, or we find a different service entirely.